From bf8542ee508e4a15b810a65811d4065496a822cc Mon Sep 17 00:00:00 2001
From: Vlastimil Babka <vbabka@suse.cz>
Date: Fri, 19 Mar 2021 18:56:50 +0100
Subject: [PATCH 15/15] selftests: add a kselftest for SLUB debugging
 functionality-fix

Remove a BUILD_BUG_ON left over from original resiliency_test() that breaks
builds.

Reported-by: Naresh Kamboju <naresh.kamboju@linaro.org>
Signed-off-by: Vlastimil Babka <vbabka@suse.cz>
Link: https://lore.kernel.org/r/53cc267d-aa0a-071a-f9f8-0b47ec4b2b9d@suse.cz/
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 lib/test_slub.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/lib/test_slub.c b/lib/test_slub.c
index 0075d9b44251..b7ad6c0b07a6 100644
--- a/lib/test_slub.c
+++ b/lib/test_slub.c
@@ -97,9 +97,6 @@ static void __init test_clobber_redzone_free(void)
 
 static void __init resiliency_test(void)
 {
-
-	BUILD_BUG_ON(KMALLOC_MIN_SIZE > 16 || KMALLOC_SHIFT_HIGH < 10);
-
 	pr_err("SLUB resiliency testing\n");
 	pr_err("-----------------------\n");
 	pr_err("A. Corruption after allocation\n");
-- 
2.31.0

